rasa/core/tracker_store.py

```diff
@@ -359,7 +359,8 @@ def _set_key_prefix(self, key_prefix: Text) -> None:
             self.key_prefix = key_prefix + ":" + DEFAULT_REDIS_TRACKER_STORE_KEY_PREFIX
         else:
             logger.warning(
-                f"Omitting provided non-alphanumeric redis key prefix: '{key_prefix}'. Using default '{self.key_prefix}' instead."
+                f"Omitting provided non-alphanumeric redis key prefix: '{key_prefix}'."
+                f" Using default '{self.key_prefix}' instead."
```

Comment:
just for consistency with the other ones where we put the space at the end
```suggestion
                f"Omitting provided non-alphanumeric redis key prefix: '{key_prefix}'. "
                f"Using default '{self.key_prefix}' instead."
```